engine/apps/alerts/tasks/delete_alert_group.py

```diff
@@ -10,7 +11,7 @@
 @shared_dedicated_queue_retry_task(
     autoretry_for=(Exception,), retry_backoff=True, max_retries=1 if settings.DEBUG else None
 )
-def delete_alert_group(alert_group_pk, user_pk):
+def delete_alert_group(alert_group_pk: int, user_pk: int) -> None:
```

Comment:
Maybe not for this PR, but wondering if this still needs to be a task?

Reply:
I think we can keep it for now, I guess something could go wrong in the stop escalation part of delete and we could retry here.